include/vctr/Containers/VctrBase.h

```diff
@@ -291,6 +310,58 @@ class VctrBase : private Config,
     //==============================================================================
     // Finding elements and manipulating them
     //==============================================================================
+    /** Returns an iterator to the first element that equaled valueToLookFor or end() if none was found */
+    auto find (const ElementType& valueToLookFor)
+    {
+        return std::find (begin(), end(), valueToLookFor);
+    }
+
+    /** Returns a const iterator to the first element that equaled valueToLookFor or end() if none was found */
+    auto find (const ElementType& valueToLookFor) const
+    {
+        return std::find (begin(), end(), valueToLookFor);
+    }
+
+    /** Returns a reverse iterator to the last element in this vector that equaled p or rend() if none was found */
+    auto findReverse (const ElementType& valueToLookFor)
+    {
+        return std::find (rbegin(), rend(), valueToLookFor);
+    }
+
+    /** Returns a const reverse iterator to the last element in this vector that equaled p or rend() if none was found */
+    auto findReverse (const ElementType& valueToLookFor) const
+    {
+        return std::find (rbegin(), rend(), valueToLookFor);
+    }
+
+    /** Returns an iterator to the first element in this vector for which pred returned true or end() if none was found */
+    template <is::functionWithSignatureOrImplicitlyConvertible<bool (const ElementType&)> F>
+    auto findIf (F&& pred)
+    {
+        return std::find_if (begin(), end(), pred);
```

Comment:
So no need for `std::forward<F> (pred)` here because it is the only occurrence of the variable?

Reply:
Oh no, I missed forwarding it perfectly here. Thanks for the heads-up